playwright.config.ts

```diff
@@ -22,6 +22,14 @@ const TIMEOUT = 5 * 60 * 1000;
 export default defineConfig({
 	timeout: TIMEOUT,
 	workers: DEV ? 5 : 2,
+	expect: {
+		toHaveScreenshot: {
+			threshold: 0.25,
+			maxDiffPixelRatio: 0.025,
+			animations: 'disabled',
+			caret: 'hide'
```

Comment:
as discussed offline, let's not make the tests more lax, let's keep them as they are and focus to check where and how the differences arises